app/assets/stylesheets/main.css.scss

```diff
@@ -52,3 +52,6 @@ $compatibility-high-color: #84C93F;
 @import 'styles/pages/home';
 @import 'styles/pages/styles';
 
+// Import modular CSS
+@import "modules/**/**/*";
+@import "modules/**/**/**/*";
```

Comment:
@simonv3 @warpling I just tested it; by jove, you're right! Sure I had an issue with a ruby app some time ago, probably fixed globbing when they baked it into Rails 4.